tests/test_batch_result.py

```diff
@@ -1,11 +1,13 @@
 import json
-from pathlib import Path
-
-import xarray
 import pytest
-
+import xarray
 from openeo.metadata import Band
-from openeogeotrellis.deploy.batch_job import extract_result_metadata,run_job
+from pathlib import Path
```

Comment:
I guess this was done automatically with a tool,
 but pathlib is from stdlib, so should not be moved under pytest, xarray, ...